s3/templates/log-data.json.tpl

```diff
@@ -22,7 +22,9 @@
       "Action": "s3:*",
       "Effect": "Allow",
       "Principal": {
-        "AWS": "arn:aws:iam::${account_id}:role/TDRLogDataCrossAccountRoleMgmt"
+          "AWS": [
+            "arn:aws:iam::${external_account_1}:role/TDRLogDataAssumeRoleIntg"
```

Comment:
Should this role be hardcoded as "...Intg". Should there either be a management role ("...Mgmt"), or an individual role for each environment?

Reply:
You're correct - there needs to be a role per environment (except for management where it's not required). However if I try and apply this before the role exists, it errors - I assume AWS needs the role to be present before it's included in a bucket policy. So my plan is to leave the code as it is for now, deploy to staging and production via Jenkins, then update to include the staging and production roles in the bucket policy, and apply.

Reply:
Ahh, OK.